docs/docs/build/examples/metamask-ordinals/index.md

```diff
@@ -0,0 +1,80 @@
+# Use MetaMask to Inscribe and Transfer Ordinals, and Transfer BTC
+
+This example demonstrates how users can use a single web wallet to manage assets on both Bitcoin and BOB. The BOB MetaMask Snap can be used to inscribe and transfer ordinals, and transfer BTC, using only MetaMask as well as access any other EVM functions on BOB.
+
+:::info Example Code
+
+The code for this example can be [found in the GitHub repository](https://github.com/bob-collective/demo-brc20-metamask)
```

Comment:
[not blocking] nitpick: I'd suggest using the active voice ("Find the code for this example on GitHub here") rather than passive.

Reply:
Let's change that in a follow up PR for all the demos.